src/screens/styles.ts

```diff
@@ -5,6 +5,68 @@ export const styles = StyleSheet.create({
     flex: 1,
     flexDirection: 'column',
     alignItems: 'center',
+    backgroundColor: 'grey',
+    gap: 1,
+  },
+  post: {
+    paddingHorizontal: 15,
+    paddingTop: 8,
     backgroundColor: 'white',
   },
+  person_header: {
+    paddingTop: 15,
+    alignSelf: 'stretch',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  name_profile: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: 10,
+  },
+  username: {
+    fontWeight: 'bold',
+    fontSize: 13,
+  },
+  post_date: {
+    color: 'grey',
+    paddingRight: 2,
+    fontSize: 11,
+  },
+  da_text: {
+    paddingVertical: 15,
+    fontSize: 13,
+  },
+  da_link: {
+    color: 'blue',
+    textDecorationLine: 'underline',
+  },
+  da_image: {
+    width: '100%',
+    height: undefined,
+    aspectRatio: 1.5,
+    borderRadius: 10,
+  },
```

Comment:
Try to be more concise with your variable names so it's easier for your fellow devs to read/review code!

Also, we use the `camelCase` naming convention for react native CSS.